### ISSUE TITLE
HTML Authentication option missing in v3?

### ISSUE THREAD

**Issue by TychoSchenkeveld** [browserpass/browserpass-native#53]
I just got the update for version 3 in my browser. It looks a lot better now, thanks! I did have to upgrade the native messaging host, but I was aware of this because I saw a pop-up notification flash by a few days ago. Thanks for that!

Anyway, I just wanted to mention that the feature to fill in HTML authentication now seems missing? Previously this was a globe icon. I still need this for some websites, are you planning on bringing it into v3 later? Or was it removed because of the way it worked? (You had to launch the website through the extension of course, wasn't an issue for me, it worked well anyway).

PS: It would be good to mention on the Readme page that it's not enough to just replace the new EXE with the old one. You also have to update the registry with the old "dannykooten" keys to the new ones and use the new JSON files :) I thought I'd be smart and just overwrite the EXE but that didn't work for that reason.

**Comment by maximbaz** [comment]
Hey! The functionality is not removed (we still have to deal with modal HTTP auth), but it is only provided via keyboard shortcuts now (`Ctrl+G` and `Ctrl+Shift+G`), a globe button was removed because it confused many users who tried to use Browserpass as a bookmark manager, and also because very few websites actually use HTTP auth (majority use normal login forms), so this button doesn't warrant to be visible at all times in popup.

**Comment by TychoSchenkeveld** [comment]
Oh brilliant, keyboard shortcuts are fine. Very happy it's still in there.

And indeed, very few websites use it.. It's just a few dinosaurs in work that use it :P

**Comment by maximbaz** [comment]
I'm glad you saw the notification a few days ago (so this upgrade didn't catch you by surprise) and that you were able to perform a reasonably painless upgrade 😄 

It is true that registry needs to be updated, I intended to highlight this by putting "refer to the section Configure browsers" in bold 😉 

yubikey-touch-detector sadly won't work on Windows, there's no official way to detect whether a yubikey is waiting on you, so I just found a bunch of hacks that magically happened to work on Linux, but they rely very much on Linux file system and file structure...

**Comment by maximbaz** [comment]
(moving to browserpass-extension repo and closing, feel free to create a new issue if you stumble upon something else).
